micro-benchmarks/matmul_relu.py

```diff
@@ -0,0 +1,84 @@
+import sre_compile
+import torch
+from benchmark_helper import time_with_torch_timer
+from torchinductor.triton_ops.matmul import matmul_out as triton_mm_out
+from torchinductor.triton_ops.matmul import matmul as triton_mm
+import torchdynamo
+import torchinductor.config as config
+
+config.triton.use_mm = True
+
+@torchdynamo.optimize("inductor", nopython=True)
+def inductor_mm(a, b):
+    return torch.mm(a, b)
+
+def torch_mm_relu(a, b):
+    return torch.nn.functional.relu(torch.matmul(a, b))
+
+def torch_mm(a, b):
+    return torch.matmul(a, b)
+
+
+if __name__ == "__main__":
+    # Real shapes from torchbench
+    a_shapes = [ [2048, 768], [64, 1280], [2048, 768], [32, 2048], [1, 39200], [128, 3072], [16, 1280]]
+    b_shapes = [ [768, 3072], [1280, 1000], [768, 768], [2048, 1000], [39200, 50], [3072, 1000], [1280, 1000]]
+
+    # Artificial larger shapes
+    a_shapes += [[10240, 512], [10240, 1024]]
+    b_shapes += [[512, 10240], [1024, 10240]]
+
+    for i in range(len(a_shapes)):
+        a_shape = a_shapes[i]
+        b_shape = b_shapes[i]
+        print("Shape:", a_shape, 'x', b_shape)
+        a = torch.randn(a_shape, device='cuda', dtype=torch.float16)
+        b = torch.randn(b_shape, device='cuda', dtype=a.dtype)
+
+        time_with_torch_timer(torch_mm, (a, b), string_id="torch mm")
+        time_with_torch_timer(torch_mm_relu, (a, b), string_id="torch mm + relu")
+        time_with_torch_timer(inductor_mm, (a, b), string_id="inductor mm")
+        
+
+
+
+
+# Results preview
```

Comment:
minor nit: Can you throw a little bit of hardware spec on here? For example, looking at a random comment around benchmark timing in torchbench.py:

`# These benchmarks took >600s on an i9-11900K CPU`

Reply:
Hi Michael, thanks a lot for the comments! The experiments were run on the AWS AI cluster. Will add the CPU and GPU info there.